tests/test_inline.py

```diff
@@ -15,7 +15,9 @@ def basic():
             return 1
 
     inlined = apply_passes([if_inline()])(basic)
-    assert inspect.getsource(inlined) == f'''\
+    inlined_src = inspect.getsource(inlined)
+    print(inlined_src)
+    assert inlined_src == f'''\
 def basic():
     return {0 if cond else 1}
 '''
```

Comment:
```suggestion
''', inlined_src
```

Reply:
Could make `inlined_src` the assertion message, so it only prints when there's an error

Reply:
Was just a stray print from debugging, also pytest still wouldn't pretty print the newline which was the my goal with print